joomla-extensions/weblinks
# Review XML CS for the XML files in the component

### Summary of Changes

Review XML CS for the XML files in the component

### Testing Instructions

Confirm the admin & Frontend component options still work (for the frontend it is the submit form)

### Expected result

Use cs

### Actual result

Use not cs

### Documentation Changes Required

None.
